semgrep/semgrep/rule_schema.yaml

```diff
@@ -49,6 +50,37 @@ definitions:
     required:
     - metavariable-regex
     additionalProperties: false
+  metavariable-pattern:
```

Comment:
Does this accept also just pattern: like
```
metavariable-pattern:
  metavariable: $OPTS
  pattern: foo(...)
```
?

Reply:
In semgrep-core it does because I just reuse the same parsing function as for top-level formulas. However in Python I allowed only `patterns` and `pattern-either` to keep it short. I imagined these were the most useful cases. Now that you are asking for it and I give it another thought, I do see useful cases for `pattern`. Let's add it.